include/RAJA/policy/cuda/kernel/CudaKernel.hpp

```diff
@@ -75,7 +77,7 @@ using cuda_explicit_launch = cuda_launch<async0, num_blocks, num_threads, blocks
  * If num_threads is 0 then num_threads is chosen at runtime.
  */
 template <size_t num_threads0, bool async0>
-using cuda_occ_calc_launch = cuda_launch<async0, 0, num_threads0, 0>;
+using cuda_occ_calc_launch = cuda_explicit_launch<async0, 0, num_threads0, 0>;
```

Comment:
I believe blocks_per_sm should default to 1 instead of 0.

Reply:
Done, thanks!